Gemfile

```diff
@@ -36,24 +36,27 @@ gem 'auth-armor'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'database_cleaner'
+  gem 'factory_bot_rails'
+  gem 'rspec-rails', '~> 3.5'
+  gem 'rubocop', '~> 0.49.1',   require: false
+  gem 'shoulda-matchers'
+  gem 'dotenv-rails', '~> 2.0.1'
+  gem 'faker', :git => 'https://github.com/stympy/faker.git', :branch => 'master'
+  gem 'pry'
```

Comment:
you don't need pry if you have pry-rails because it is already included in the gem